x-pack/plugin/rank-rrf/src/test/java/org/elasticsearch/xpack/rank/linear/LinearRetrieverBuilderParsingTests.java

```diff
@@ -43,6 +43,16 @@ public static void afterClass() throws Exception {
     @Override
     protected LinearRetrieverBuilder createTestInstance() {
         int rankWindowSize = randomInt(100);
+
+        List<String> fields = null;
+        String query = null;
+        ScoreNormalizer normalizer = null;
+        if (randomBoolean()) {
+            fields = randomList(1, 10, () -> randomAlphaOfLengthBetween(1, 10));
+            query = randomAlphaOfLengthBetween(1, 10);
+            normalizer = randomScoreNormalizer();
```

Comment:
If we don't return here doesn't that create an invalid linear retriever with mixed fields and inner retrievers?

Reply:
Ok thanks for explaining, let's just add a comment then